Pearson-Advance/openedx-themes
# PADV-508: Add pearson-aws-theme

## Ticket

https://agile-jira.pearson.com/browse/PADV-509

## Description

This PR adds the templates and assets for pearson-aws-theme on edx-platform.


## Testing:

- Enable comprehensive theming.
- Run the LMS: `make dev.up.lms`.
- Set `pearson-aws-theme` on a site.
- Compile the AWS theme.
- Verify that the AWS theme is applied correctly on Olive.

## Reviewers
